docs/js/installer/github.js

```diff
@@ -0,0 +1,143 @@
+/* global fetch */
+const githubUrl = 'https://api.github.com';
```

Comment:
I added the `config` module in #45. Maybe could we add those const in the `config/default.js` ?